src/Tribe/Attendee_Registration/View.php

```diff
@@ -213,41 +45,14 @@ public function display_attendee_registration_shortcode() {
 	 * @return bool|string
 	 */
 	public function get_cart_url( $provider ) {
-		if ( is_numeric( $provider ) ) {
-			/** @var Tribe__Tickets__Tickets_Handler $tickets_handler */
-			$tickets_handler = tribe( 'tickets.handler' );
-			$provider        = get_post_meta( absint( $provider ), $tickets_handler->key_provider_field, true );
-		}
-
-		if ( empty( $provider ) ) {
-			return false;
-		}
-
-		$post_provider = $this->get_cart_provider( $provider );
-
-		if ( empty( $post_provider ) ) {
-			return false;
-		}
-
 		try {
-			if ( 'Tribe__Tickets_Plus__Commerce__WooCommerce__Main' === get_class( $post_provider ) ) {
-				/** @var \Tribe__Tickets_Plus__Commerce__WooCommerce__Main $provider */
-				$provider = tribe( 'tickets-plus.commerce.woo' );
-			} elseif ( 'Tribe__Tickets_Plus__Commerce__EDD__Main' === get_class( $post_provider ) ) {
-				/** @var \Tribe__Tickets_Plus__Commerce__EDD__Main $provider */
-				$provider = tribe( 'tickets-plus.commerce.edd' );
-			} else {
-				return;
-			}
-		} catch ( RuntimeException $exception ) {
-			return;
-		}
+			/* @var Tribe__Tickets_Plus__Attendee_Registration__View $view */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: 010ae28bf6c952236403d8ca86822b17</sub>